curriculum/challenges/english/01-responsive-web-design/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-002.md

```diff
@@ -0,0 +1,78 @@
+---
+id: 6193da99ca92051a07fb8a13
+title: Step 2
+challengeType: 0
+dashedName: step-2
+---
+
+# --description--
+
+Within your `body` element, create a `div` element with the `id` set to `sheet`. Below that, create a `footer` element with the text set to `Last Updated: August 2021`.
```

Comment:
```suggestion
Within your `body` element, create a `div` element with the `id` set to `sheet`. Below that, create a `footer` element with the text set to `Last Updated: December 2021`.
```

Reply:
lol - yea, we're gonna need to update this every month